combat-trainer.lic

```diff
@@ -607,20 +610,21 @@ class SpellProcess
   def check_charging?(game_state)
     return false unless game_state.charge_cambrinth?
 
-    find_cambrinth3(@cambrinth, @held_cambrinth)
```

Comment:
Yes. I moved `find_cambrinth` and `stow_cambrinth` outside of the if. Make sure you don't revert the change in this PR.